src/modules/navigation/MainNavigator.js

```diff
@@ -12,91 +12,68 @@ const Stack = createStackNavigator();
+          },
+          headerTintColor: Colors.WHITE,
+          headerTitleStyle: {
+            fontSize: Typography.FONT_SIZE_17,
+            alignSelf: "center",
+            color: Colors.PURPLE_VERY_LIGHT,
+            fontFamily: Typography.FONT_FAMILY_BOLD,
+            paddingBottom: 7,
+          },
+          headerRight: () => (
+            <FontForgeIcon
+              name="doctor_profile"
+              size={30}
+              color={Colors.PURPLE_VERY_LIGHT}
+              style={{
+                marginRight: 20,
+                marginBottom: 7,
+              }}
+            />
+          ),
+          headerBackImage: () => (
+            <FontForgeIcon
+              name="back"
+              size={32}
+              color={Colors.PURPLE_VERY_LIGHT}
+              style={{
+                paddingTop: 8,
+                marginLeft: 11,
+                alignSelf: "flex-start",
+                fontWeight: Typography.FONT_WEIGHT_BOLD,
+                transform: [{ rotate: "352deg" }],
+              }}
+            />
+          ),
+        }}
+      >
         <Stack.Screen
           name="PatientsList"
           component={PatientsListScreen}
-          options={{ title: "Patients list" }}
+          options={{ title: "Lista pacjentów" }}
         />
         <Stack.Screen
           name="PatientCard"
           component={PatientCardScreen}
           options={{
             title: "Karta pacjenta",
-            headerStyle: {
-              backgroundColor: Colors.PURPLE,
-              height: 105,
-              borderBottomLeftRadius: 50,
-              elevation: 0,
-              shadowOpacity: 0,
-            },
-            headerTintColor: Colors.WHITE,
-            headerTitleStyle: {
-              fontSize: Typography.FONT_SIZE_17,
-              alignSelf: "center",
-              color: Colors.PURPLE_VERY_LIGHT,
-              fontFamily: Typography.FONT_FAMILY_BOLD,
-              paddingBottom: 7,
-            },
-            headerBackImage: () => (
-              <FontForgeIcon
-                name="back"
-                size={32}
-                color={Colors.PURPLE_VERY_LIGHT}
-                style={{
-                  paddingTop: 8,
-                  marginLeft: 11,
-                  alignSelf: "flex-start",
-                  fontWeight: Typography.FONT_WEIGHT_BOLD,
-                  transform: [{ rotate: "352deg" }],
-                }}
-              />
-            ),
-            headerRight: () => (
-              <FontForgeIcon
-                name="doctor_profile"
-                size={30}
-                color={Colors.PURPLE_VERY_LIGHT}
-                style={{
-                  marginRight: 20,
-                  marginBottom: 7,
-                }}
-              />
-            ),
           }}
         />
         <Stack.Screen
           name="AddPatient"
           component={AddPatientScreen}
           options={{
             title: "Dane osobowe",
```

Comment:
w tym ekranie nie ma być strzałki wstecz w headerze, w kodzie odpowiada za to 
`headerLeft: null,`

Reply:
A faktycznie, nie ma tego w widoku, ale mimo wszystko bym ją zostawił. Wydaj mi się, że dla lekarzy będzie to bardziej intuicyjnej, bo i tak mogą wycofać dodanie pacjenta przez gest/przycisk na telefonie.

Reply:
Jak dla mnie też może zostać, ale może wspomnijmy o tym jutro na spotkaniu online bo chyba był jakiś powód dla którego strzałka miała być wycofana, natomiast ja nie pamiętam jaki

Reply:
Oki to jutro to jeszcze poruszymy :)